docs/setParameter_tuning.md

```diff
@@ -0,0 +1,100 @@
+# Percona Server for MongoDB <br/> setParameter tuning guide
+
+There are several setParameter options that can be used to
+adjust the operation of Percona Server for MongoDB.
+
+## cursorTimeoutMillis
+
+Command line:
+
+```
+--setParameter cursorTimeoutMillis={integer}
+```
+
+Configuration File:
+
+```
+setParameter:
+  cursorTimeoutMillis: {integer}
+```
+
+Default value: 600000 (Ten Minutes)
+
+Sets the duration of time after which idle query cursors will be removed from
+memory.
+
+## failIndexKeyTooLong
+
+Command Line:
+```
+--setParameter failIndexKeyTooLong={true|false}
+```
+
+Configuration File:
+```
+setParameter:
+  failIndexKeyTooLong: {true|false}
+```
+
+Default value: true 
+
+Versions of MongoDB prior to 2.6 would insert and update documents even if an
+index key was too long.  The documents would not be included in the index.
+Newer versions of MongoDB will not insert or update the documents with the
+failure.  By setting this value to false, the old behavior is enabled.
+
+## internalQueryPlannerEnableIndexIntersection
+
+Command Line:
+```
+--setParameter internalQueryPlannerEnableIndexIntersection={0|1}
```

Comment:
This parameter is defined in code as accepting `bool` values `{true|false}`
It will also accept `{1|0}` of course.